fiftyone/core/state.py

```diff
@@ -239,10 +239,20 @@ def serialize_fields(schema: t.Dict) -> t.List[SampleField]:
                     embedded_doc_type=embedded_doc_type,
                     subfield=subfield,
                     description=field.description,
-                    info=dict(**field.info)
+                    info=_convert_mongoengine_data(field.info)
                     if isinstance(field.info, BaseDict)
                     else field.info,
                 )
             )
 
     return data
+
+
+def _convert_mongoengine_data(data):
+    if isinstance(data, BaseDict):
+        return {k: _convert_mongoengine_data(v) for k, v in data.items()}
```

Comment:
this will make ordered dicts into unordered dicts. not sure what implications if any that has, but just wanted to call it out in case

Reply:
shouldn't be an issue for python3.7 and after

Reply:
JSON data is not ordered, and this is only for sending payload data